torch/_dynamo/config.py

```diff
@@ -421,6 +421,14 @@ def default_debug_dir_root():
     if DEBUG_DIR_VAR_NAME in os.environ:
         return os.path.join(os.environ[DEBUG_DIR_VAR_NAME], "torch_compile_debug")
     elif is_fbcode():
+        fbcode_dir = os.path.join(os.path.expanduser("~"), "fbsource", "fbcode")
```

Comment:
Is this path general?  Should this be cwd or home?  Putting it in the fbsource tree seems a bit weird.

Reply:
It's intended to be in the fbcode. The generated TARGETS file and python files need to be in fbsource for them to be compiled by buck.

Reply:
I guess I could also make it cwd.